src/qt/stattext.cpp

```diff
@@ -55,6 +55,34 @@ bool wxStaticText::Create(wxWindow *parent,
     m_qtLabel->setBuddy( m_qtLabel );
     m_qtLabel->setTextInteractionFlags( Qt::NoTextInteraction );
 
+    Qt::AlignmentFlag qt_alignment = Qt::AlignAbsolute;
+
+    switch (style & wxALIGN_MASK)
```

Comment:
Using `switch` here is not a good idea because the style can be a combination of bits, e.g., if vertical alignment is supported, it could be `wxALIGN_RIGHT | wxALIGN_BOTTOM`. The other ports just use a sequence of `if`s, in the right order (because `wxALIGN_LEFT` is 0!), see e.g. `wxStaticText::MSWGetStyle()` in `src/msw/stattext.cpp`.